src/database/database.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+#include <cstdint>
+#include <iostream>
+#include <vector>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
```

Comment:
remove unused includes. You can add these includes in future, when will be needed

Reply:
@KuWojtow, please take care of it

Reply:
I removed checking the IP of the user who connects to the database.